views/about.tmpl

```diff
@@ -62,6 +62,9 @@
           <h3>AM Radio</h3>
           <p>We are of course a legal radio station and we broadcast on AM radio all across campus on 1350KHz Medium Wave.</p>
           <p>So what are you waiting for? Ask your parents if you can have their old boom box, and get it tuned into URY like it’s 1989.</p>
+          <h3>FM Radio</h3>
+          <p>We're also braodcasting across campus on 88.3MHz FM.</p>
+          <p>Did your parents get rid of the boom box at a car boot sale? Pick us up on that radio you got yourself because your at uni and you're an "adult" now.</p>
```

Comment:
your. (You use it correctly 4 words later... Impressive)

Reply:
i did it wrong for the second one originally, and then went back and fixed it 😂